collect_app/build.gradle

```diff
@@ -160,6 +160,12 @@ dependencies {
     androidTestCompile group: 'com.android.support', name: 'support-annotations', version: '25.1.0'
     androidTestCompile group: 'com.android.support.test', name: 'runner', version: '0.5'
     androidTestCompile group: 'com.android.support.test', name: 'rules', version: '0.5'
+
+    // firebase core library
```

Comment:
I'm sorry, I didn't notice before that this was after the test dependencies! These should be up with the other non-test dependencies. And I'm not sure you really need the comments since the names of the libraries are so clear.